newIDE/app/src/EventsBasedObjectEditor/index.js

```diff
@@ -116,6 +116,14 @@ export default function EventsBasedObjectEditor({
           onChange();
         }}
       />
+      <Checkbox
+        label={<Trans>Expand inner area with parent</Trans>}
+        checked={eventsBasedObject.isInnerAreaFollowingParentSize()}
+        onCheck={(e, checked) => {
+          eventsBasedObject.markAsInnerAreaExpandingWithParent(checked);
+          onChange();
+        }}
+      />
```

Comment:
Let's make a Select that is either "Expand (or reduce) inner area when the parent size changes" 
or "Keep inner area (and stretch the rendered object) when the parent size changes".

Can be in another PR.